_posts/2024-05-17-Parental-Leave-Policy.md

```diff
@@ -0,0 +1,59 @@
+
+None of the above, established and “typical” benefits, worked for the situation Compiler faced. The company needed to create its own parental leave program from scratch and wanted to do so in a way that treated all of its employees, regardless of their residence, excellently and with dignity.
+
+Like most folks starting a project they are novices in, I Googled. I read articles from Kaiser Family Foundation, the Center for American Progress, Think Babies, and even watched an episode of Full Frontal with Samantha Bee on the topic of paid parental leave. SHRM, the Society for Human Resources Management was a great resource for sample policies and language. I also read articles from California’s Employment Development Department because Compiler is a California-based LLC. I researched information to inform both the company’s future parental leave policy and the policy I’d be the first recipient of. The sheer volume of information, however, was overwhelming, especially for a small, young company with employees in multiple states.
+
+Fortunately, Compiler is a member of a PEO (professional employer organization). I reached out to them, and they provided sample leave policies, guidance, and support, but frequently emphasized the importance of seeking legal counsel.
+
+After reading multiple sample policies and language from SHRM, the PEO, and elsewhere, I took my favorite parts from each and patched them together to create Compiler’s first draft of a parental leave policy. Of highest importance was, and is:
+
+There be no tenure requirement; leave and wage replacement be available immediately to all full-time employees
+- Offer the same parental leave benefits to everyone, regardless of where they live. Identify the state with the requirements friendliest to the employee, and apply them to all employees
+- Leave be at least 12 weeks long, the amount of time as the “fourth trimester”
+- Provide at least 50% of the employee’s regular wages during the course of the leave
+
+There was a lot happening at Compiler leading up to my due date. Many spreadsheets and forecasts and models were created to estimate how much Compiler could afford to pay someone to be away from work for X, Y, or Z number of weeks. It was tense. Compiler was a young company, and frankly, Vyki was not yet totally bought into paid parental leave, and was understandably cautious about the financial impacts. I loosely dreamed of a lofty 3-6 month leave at 100% of my pay. I quietly presented that as one of several options the company could adopt, but ultimately, and understandably, we started more modestly.
```

Comment:
@vykster wanted to run this paragraph by you. Let us know if you'd like it to be edited, removed entirely, or posted as is. Specifically this sentence: `Compiler was a young company, and frankly, Vyki was not yet totally bought into paid parental leave, and was understandably cautious about the financial impacts.`